app/middleware/errors.js

```diff
@@ -2,6 +2,15 @@ const sentry = require('raven');
 const errlog = require('bole')('error-handler');
 
 
+const handle_state_mismatch = (req, res, conf) => {
```

Comment:
This should be `function handle_state_mismatch(req, res, conf) {`

Reply:
I've seen both being used but I guess in this case there is no reason to stick the function in a constant really, I'll change 👍

Reply:
Here's a good couple of reasons to prefer `function`, at least at the top level of the module: https://medium.freecodecamp.org/constant-confusion-why-i-still-use-javascript-function-statements-984ece0b72fd